assets/main.js

```diff
@@ -1,48 +1,48 @@
 // document query .signup-button
-
 const signup_button = document.querySelector('.signup-button')
 const signup_input = document.querySelector('.signup-input')
-console.log('signup_button', signup_button)
```

Comment:
This whole section which has been "removed" was just wrapped in a `if (signup_button)` condition to fix console error about missing element.